packages/doenetml/src/state/slices/main.ts

```diff
@@ -92,7 +92,9 @@ export const mainThunks = {
                 const updatesToIgnore = updatesToIgnoreRef.current;
 
                 if (updatesToIgnore.size > 0) {
-                    let valueFromRenderer = updatesToIgnore.get(actionId || "");
+                    let valueFromRenderer = updatesToIgnore.get(
+                        `${actionId}|${componentIdx}`,
```

Comment:
I thought this change appeared in 3 other places?

Reply:
Yeah, my bad. I checked in the wrong version where I was testing that the test broke when I reverted! It is fixed now.